.projenrc.ts

```diff
@@ -24,9 +23,10 @@ const project = new clickupCdk.ClickUpCdkConstructLibrary({
     'sinon-spy-utils',
   ],
   peerDeps,
+  projenrcTs: true,
 
+  author: '', // leverage default
```

Comment:
Is a required var, but confirmed we still see the default `ClickUp` value in `package.json`. This is abstracted by `clickup-projen`.